node-red-project/flows.json

```diff
@@ -9372,5 +9380,521 @@
+            }
+        ],
+        "action": "",
+        "property": "",
+        "from": "",
+        "to": "",
+        "reg": false,
+        "x": 490,
+        "y": 440,
+        "wires": [
+            [
+                "c497cfedf6aed533"
+            ]
+        ]
+    },
+    {
+        "id": "f894a3a8017a7de4",
+        "type": "function",
+        "z": "f410f6aac9d1a3fc",
+        "name": "set up tb metrics post",
+        "func": "// The server URL, port, and device access token must be stored in the config context\n// before this will work\nlet serverUrl = global.get(\"serverUrl\", \"config\")\nlet serverPort = global.get(\"serverPort\", \"config\")\nlet accessToken = global.get(\"accessToken\", \"config\")\nmsg.url = `${serverUrl}:${serverPort}/api/v1/${accessToken}/telemetry`\nmsg.method = \"POST\";\nreturn msg;",
+        "outputs": 1,
+        "timeout": 0,
+        "noerr": 0,
+        "initialize": "",
+        "finalize": "",
+        "libs": [],
+        "x": 500,
+        "y": 600,
+        "wires": [
+            [
+                "634f581fd150b60c"
+            ]
+        ]
+    },
+    {
+        "id": "521115ddd5741eea",
+        "type": "function",
+        "z": "f410f6aac9d1a3fc",
+        "name": "format metrics payload",
+        "func": "// format the payload with a timestamp\nlet values = {\n    ...msg.payload,\n}\n//ts in milliseconds\nmsg.payload = {\n    ts: Date.now(),\n    values\n}\n\nreturn msg;",
+        "outputs": 1,
+        "timeout": 0,
+        "noerr": 0,
+        "initialize": "",
+        "finalize": "",
+        "libs": [],
+        "x": 260,
+        "y": 600,
+        "wires": [
+            [
+                "452b53218521f20f",
+                "f894a3a8017a7de4"
+            ]
+        ]
+    },
+    {
+        "id": "e33728beb3e15199",
+        "type": "debug",
+        "z": "f410f6aac9d1a3fc",
+        "name": "metrics post response",
+        "active": true,
+        "tosidebar": true,
+        "console": false,
+        "tostatus": false,
+        "complete": "payload",
+        "targetType": "msg",
+        "statusVal": "",
+        "statusType": "auto",
+        "x": 1020,
+        "y": 600,
+        "wires": []
+    },
+    {
+        "id": "634f581fd150b60c",
+        "type": "http request",
+        "z": "f410f6aac9d1a3fc",
+        "name": "tb telemetry metrics http api",
+        "method": "use",
+        "ret": "txt",
+        "paytoqs": "ignore",
+        "url": "",
+        "tls": "",
+        "persist": false,
+        "proxy": "",
+        "insecureHTTPParser": false,
+        "authType": "",
+        "senderr": false,
+        "headers": [],
+        "x": 760,
+        "y": 600,
+        "wires": [
+            [
+                "e33728beb3e15199"
+            ]
+        ]
+    },
+    {
+        "id": "c0078fa5b3946c05",
+        "type": "exec",
+        "z": "f410f6aac9d1a3fc",
+        "command": "python3 ./traffic-monitor/utils/rpi5-nodered-telemetry.py",
```

Comment:
Just wanted to get your take:  How much system impact does this have being run every 5-minutes?  It spawns a new python3 process and then it ends after it returns the values, is that right?

Reply:
Runs one time, outputs the result.  
Adapted from the other script that DID run via console.